packages/h5wasm/src/h5wasm-api.ts

```diff
@@ -102,6 +113,8 @@ export class H5WasmApi extends ProviderApi {
 
       const children = h5wEntity.keys().map((name) => {
         const h5wChild = h5wEntity.get(name);
+        assertNonNull(h5wChild);
```

Comment:
Perhaps worth putting a more meaningful error message as in `getH5WasmEntity` ?

Reply:
Arf just saw your comment https://github.com/silx-kit/h5web/pull/1082#discussion_r858522417. Discard this comment then.